pandas/tests/dtypes/test_dtypes.py

```diff
@@ -115,6 +115,11 @@ def test_construction_from_string(self, dtype):
         with pytest.raises(TypeError, match=msg):
             CategoricalDtype.construct_from_string("foo")
 
+    def test_pyarrow_string_import_error(self, dtype):
+        # GH-44276
+        assert not is_dtype_equal(dtype, "string[pyarrow]")
+        pytest.importorskip("pyarrow")
```

Comment:
use the decorator on the function itself to skip

Reply:
Done